components/tag-list/tag-list-item-mixin.js

```diff
@@ -131,52 +131,55 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 		this.clearable = false;
 		/** @ignore */
 		this.keyboardTooltipItem = false;
-		/** @ignore */
-		this.role = 'listitem';
```

Comment:
Can the property definition for `role` go away?

Reply:
Yep, thanks!